package.json

```diff
@@ -94,7 +95,7 @@
     "@mui/icons-material": ">= 5.5.1 < 6",
     "@mui/material": ">= 5.5.3 < 6",
     "@mui/x-date-pickers": ">= 6.16.0 < 7",
-    "luxon": ">= 3.4.3 < 4",
+    "luxon": "^3.4.3",
```

Comment:
This should remain with the range notation as it is inside `peerDependency` section.
Add `luxon` dependency inside `devDependencies` section.